src/verify_cert.rs

```diff
@@ -550,14 +550,11 @@ const EKU_SERVER_AUTH: &[u8] = &oid!(1, 3, 6, 1, 5, 5, 7, 3, 1);
 // id-kp-clientAuth   OBJECT IDENTIFIER ::= { id-kp 2 }
 const EKU_CLIENT_AUTH: &[u8] = &oid!(1, 3, 6, 1, 5, 5, 7, 3, 2);
 
-fn loop_while_non_fatal_error<'a, V: 'a>(
+fn loop_while_non_fatal_error<'a, V: 'a + IntoIterator>(
```

Comment:
`'a + IntoIterator` looks very strange to me, I think we usually put the lifetime(s) last?

Reply:
Fixed, thanks for calling it out.